addons/account/wizard/account_move_send.py

```diff
@@ -467,7 +467,7 @@ def _send_mails(self, moves_data):
         subtype = self.env.ref('mail.mt_comment')
         mail_template = self.mail_template_id
 
-        for move, move_data in moves_data.items():
+        for move, move_data in filter(lambda item: item[0].partner_id.email, moves_data.items()):
```

Comment:
Usually we avoid `filter` and `map` and we prefer list comprehension

Cf: https://docs.quantifiedcode.com/python-anti-patterns/readability/using_map_or_filter_where_list_comprehension_is_possible.html

Reply:
Good to know. Thank you :)